src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/UpdateInductionTest.kt

```diff
@@ -258,50 +259,298 @@ class UpdateInductionTest : IntegrationTestBase() {
+    assertThat(updatedInduction.skillsAndInterests!!.modifiedDateTime).isEqualToIgnoringNanos(initialModifiedAt)
+    assertThat(updatedInduction.inPrisonInterests!!.modifiedDateTime).isEqualToIgnoringNanos(initialModifiedAt)
+  }
+
+  @Test
+  fun `should update induction with no previous work experience`() {
+    // Given
+    val prisonNumber = aValidPrisonNumber()
+    createInduction(prisonNumber, aValidCreateCiagInductionRequest())
+    val persistedInduction = getInduction(prisonNumber)
+    val createdDateTime = persistedInduction.createdDateTime
+    val initialModifiedAt = persistedInduction.modifiedDateTime
+    val updateInductionRequest = aValidUpdateCiagInductionRequest(
+      originalInduction = persistedInduction,
+      workExperience = aValidUpdatePreviousWorkRequest(
+        id = persistedInduction.workExperience!!.id!!,
+        hasWorkedBefore = false,
+        typeOfWorkExperience = null,
+        typeOfWorkExperienceOther = null,
+        workExperience = null,
+        workInterests = aValidWorkInterests(
+          id = persistedInduction.workExperience!!.workInterests!!.id,
+          workInterests = setOf(WorkType.SPORTS),
+          workInterestsOther = null,
+          particularJobInterests = setOf(
+            WorkInterestDetail(
+              workInterest = WorkType.SPORTS,
+              role = "Football coach",
+            ),
+          ),
+        ),
+      ),
+    )
+
+    val expectedWorkExperience = aValidPreviousWorkResponse(
+      id = persistedInduction.workExperience!!.id!!,
+      hasWorkedBefore = false,
+      typeOfWorkExperience = null,
+      typeOfWorkExperienceOther = null,
+      workExperience = null,
+      workInterests = aValidWorkInterests(
+        id = persistedInduction.workExperience!!.workInterests!!.id,
+        workInterests = setOf(WorkType.SPORTS),
+        workInterestsOther = null,
+        particularJobInterests = setOf(
+          WorkInterestDetail(
+            workInterest = WorkType.SPORTS,
+            role = "Football coach",
+          ),
+        ),
+      ),
+      modifiedBy = "auser_gen",
+    )
+    // Short delay for the purpose of timestamp checking
+    Thread.sleep(500)
+
+    // When
+    webTestClient.put()
+      .uri(URI_TEMPLATE, prisonNumber)
+      .withBody(updateInductionRequest)
+      .bearerToken(aValidTokenWithEditAuthority(privateKey = keyPair.private))
+      .exchange()
+      .expectStatus()
+      .isNoContent
+
+    // Then
+    val updatedInduction = getInduction(prisonNumber)
+    assertThat(updatedInduction)
+      .hasReference(persistedInduction.reference)
+      .hasHopingToGetWork(HopingToWork.NOT_SURE)
+      .hasPrisonId("BXI")
+      .wasModifiedBy("auser_gen")
+      .wasCreatedAt(createdDateTime)
+      .wasLastModifiedAt(initialModifiedAt) // should be unchanged
+    assertThat(updatedInduction.workExperience).isEquivalentTo(expectedWorkExperience)
+    // TODO - modifiedDateTime of workExperience is not being updated
```

Comment:
I'm going to address this TODO separately. @nathanrussell-moj-digital and I know what the problem is, but we need to decide how best to tackle it.